TACC-Cloud/geoapi
# task/WG-96: support questionnaires with assets

## Overview: ##

Support assets in questionnaires.  

This PR imports the .rq file and all the associated asset images.

## Related Jira tickets: ##

* [WG-96](https://jira.tacc.utexas.edu/browse/WG-96)

## Testing Steps: ##
1. Test with related hazmapper branch https://github.com/TACC-Cloud/hazmapper/pull/144
2. Load questionnaire from the example in https://www.designsafe-ci.org/data/browser/projects/8131157524548227566-242ac117-0001-012/ "PRJ-4019 | Questionnaire_Example"
3. ensure the questionnaire is loaded
4. and can be viewed.
